src/Oakton/Descriptions/ConfigurationPreview.cs

```diff
@@ -22,8 +27,88 @@ public async Task Write(TextWriter writer)
             }
             else
             {
-                await writer.WriteLineAsync("Unable to show a preview of the configuration.");
+                await writer.WriteLineAsync(PreviewErrorMessage);
+            }
```

Comment:
We have two checks like this now. It is probably not a good idea to make `_configuration` of type `IConfigurationRoot` and throw an `AgrumentException` right in the constructor if the cast to `IConfigurationRoot` is not possible?